### DISCUSSION TITLE
GitHub PRs and commits are not being deployed.

### DISCUSSION THREAD

**Discussion by konstk1** [vercel/community#2776] [Help]
### Summary

This morning, our github commits stopped being picked up by Vercel. We can not deploy. 

### Steps to Reproduce

Submit a PR on github and it doesn't trigger build on Vercel.

**Comment by tmrclark** [comment]
Same here, push to production branch is not triggering build/deploy

**Comment by chinchang** [comment]
Same.

**Comment by tmrclark** [comment]
they just started working again for me. seems to be resolved.

**Reply by gvegayon** [reply]
Did you have to make a new PR/push?

**Reply by gvegayon** [reply]
No, mine just started using the last push

**Reply by tmrclark** [reply]
Its an issue with Github not Vercel, so Vercel will deploy last push randomly when github does whatever it needs to do to trigger it.

**Comment by konstk1** [comment]
Looks like a GitHub outage.
